kafka/admin/pom.xml

```diff
@@ -28,14 +28,6 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
         </dependency>
-        <dependency>
```

Comment:
Again, why are these somewhere replaced with `test-client-tracing` and somewhere just deleted?

Reply:
Admin had no tracing code.